src/main/java/com/onlinebanking/constant/ErrorConstants.java

```diff
@@ -0,0 +1,36 @@
+package com.onlinebanking.constant;
+
+/**
+ * Class ErrorConstants will hold all the error messages used in the application.
+ *
+ * @author George on 6/21/2021
+ * @version 1.0
+ * @since 1.0
+ */
+public enum ErrorConstants {
+
+    MISSING_REQUIRED_FIELD("Missing required field. Please check the documentation for required fields"),
+    RECORD_ALREADY_EXISTS("Record already exists"),
+    INTERNAL_SERVER_ERROR("Internal Server error"),
+    NO_RECORD_FOUND("Record with provided identifier is not found"),
+    USER_NOT_FOUND("User not found"),
+    AUTHENTICATION_FAILED("Authentication failed"),
+    COULD_NOT_UPDATE_RECORD("Could not update record"),
+    COULD_NOT_DELETE_RECORD("Could not delete record"),
+    EMAIL_ADDRESS_NOT_VERIFIED("Email address could not be verified"),
+    NOT_INSTANTIABLE("This class cannot be instantiated");
+
+    private final String errorMsg;
+
+    ErrorConstants(String errorMsg) {
+        this.errorMsg = errorMsg;
+    }
+
+    /**
+     * Constant error message.
+     * @return errorMsg String error message return from enum constants.
+     */
+    public String getErrorMsg() {
+        return errorMsg;
+    }

```

Comment:
George, you can make use of @Getter instead of creating the getErrorMsg(). Lombok can work on Enum with getter.